### PR TITLE
PS-4670 fix COMMAND_CLASS text

### PR DESCRIPTION
**Pull request by patrickbirch**
modified file:
source/management/audit log plugin
Rewrote text for Disconnect event/Query section

### CODE REVIEW

**Comment by patrickbirch** [comment]
waiting for approval or comments

**Review by nastena1606 — APPROVED** [review]
LGTM

**Pull request merged**
